inc/Ui/Ui.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace dokuwiki\Ui;
+
+/**
+ * Class Ui
+ *
+ * Abstrract base class for all DokuWiki screens
```

Comment:
Can you explain what your thinking is behind the UI class?
(Minor typo: "Abstrract")